group-generators/generators/lazybaer/index.ts

```diff
@@ -23,10 +23,25 @@ const generator: GroupGenerator = {
     const jsonListData1 = {
       "0xC96DEdb0F0972b90715FD58c75012Ca995Cb3F8E": "1",
     };
-    
+
+    const jsonListData2 = {
+      "alanred.eth": "1",
```

Comment:
Hey @cwdcwd you can add them directly in the same json
Something like that instead of having to create a new one each time
const jsonListData1 = {
      "0xC96DEdb0F0972b90715FD58c75012Ca995Cb3F8E": "1",
      "alanred.eth": "1",
      "0x34Be04572d39BC8f897e0156722898109c84F105": "1",
      "cmadison.eth": "1"
    };

Reply:
so sorry I didn't see this. I'll be sure to do that differently next time @leosayous21 ! I was just going off the existing code.